client/src/components/Button/Button.js

```diff
@@ -2,23 +2,26 @@ import React from 'react';
 import { Button as BaseButton } from 'reactstrap';
 import PropTypes from 'prop-types';
 import classnames from 'classnames';
-import IconHOC from './IconHOC';
 
-const Button = ({ className, noText, children, ...props }) =>
+const Button = ({ icon, className, noText, children, ...props }) =>
```

Comment:
This prop used to be added through the `IconHOC`.
We can't rework that HOC to add the child span in a way that's super generic, but we can easily do it here for buttons specifically - so that's what we do.